src/static/themes/greenmat/greenmat.css

```diff
@@ -35,11 +38,8 @@
 
   --tridactyl-hint-bg: rgba(0, 0, 0, 0.25); 
   --tridactyl-hint-outline: 2px solid var(--tridactyl-hintspan-bg);
-}
-
-
 
-/*custom rules over tridactyl default*/
+}
 
 :root.TridactylThemeGreenmat table {
```

Comment:
These should all get `.TridactylOwnNamespace` too or they'll apply to webcontent (the rule above applies to all tables in all pages, for example).